### DISCUSSION TITLE
Implementation Edgedb in FastApi

### DISCUSSION THREAD

**Discussion by xidoc** [fastapi/fastapi#4277] [Questions] [answered]
Hi !

Are there any plans to integrate Edgedb into FastAPi ?

**Comment by jgould22** [comment, marked as answer]
In what way would it be integrated?

It appears to have a client library(an async one!), there doesnt appear to be anything that would stop you from using it in a fastapi project.

**Reply by tiangolo** [reply]
Yep, exactly, there's nothing that needs to be done to use EdgeDB with FastAPI. They just released a video and article about it:

https://www.edgedb.com/docs/guides/tutorials/rest_apis_with_fastapi

https://www.youtube.com/watch?v=OZ_UURzDkow

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
